src/components/views/rooms/BasicMessageComposer.js

```diff
@@ -90,6 +90,7 @@ export default class BasicMessageEditor extends React.Component {
         this._modifiedFlag = false;
         this._isIMEComposing = false;
         this._hasTextSelected = false;
+        this._emoticonSettingHandle = undefined;
```

Comment:
I find it generally nicer to use `null` when explicitly assigning things and leave `undefined` for when something hasn't been assigned to anything (this would also make it consistent with above).